sympy/integrals/integrals.py

```diff
@@ -28,6 +28,8 @@
 from sympy.series.order import Order
 from sympy.series.formal import FormalPowerSeries
 from sympy.simplify.fu import sincos_to_sum
+from sympy.calculus import singularities
+from sympy import I
```

Comment:
This is not needed.

Reply:
@jksuom  I'll remove it.
Can you help me figure out why there's a problem coming in importing singularities?